lib/cog/repository/bundles.ex

```diff
@@ -17,9 +17,14 @@ defmodule Cog.Repository.Bundles do
   @reserved_bundle_names [
     Cog.embedded_bundle,
     Cog.site_namespace,
-    "user" # a bundle named "user" would break alias resolution
+    "user", # a bundle named "user" would break alias resolution
+    "cog"   # we're going to squat on this for now to prevent potential confusion
   ]
 
+  # We're also going to block the installation of any bundles that are
+  # prefixed with "cog" or the embedded bundle name
+  @reserved_bundle_name_regex Regex.compile!("^(cog|#{Cog.embedded_bundle})-.*$")
```

Comment:
You can indeed (and there are tests proving that). This just reflects the discussions we were having in chat. We could easily add other delimiters besides `-`, and possibly just block any string that starts with "operable", since I can't think of much else that legitimately starts that way. I don't know about blocking everything that starts with "cog", though, since there are a number of legitimate words that begin that way.

@imbriaco @kevsmith, do you have any thoughts on this?